test/unit/migrate_assets_to_asset_manager_test.rb

```diff
@@ -17,7 +17,7 @@ class MigrateAssetsToAssetManagerTest < ActiveSupport::TestCase
   end
 
   test 'it calls create_whitehall_asset for each file in the list' do
-    Services.asset_manager.expects(:create_whitehall_asset).with(has_entry(:file, responds_with(:path, @organisation_logo_path)))
+    Services.asset_manager.expects(:create_whitehall_asset).with(has_entry(:file, responds_with(:read, File.read(@organisation_logo_path))))
```

Comment:
It would be nice to avoid the very long line and improve the readability. Some ideas:

* Stub `Services.asset_manager` in the setup and return a mock object stored in an `@asset_manager` instance variable.
* Extract `File.read(@organisation_logo_path)` into a `content` local variable.
* Extract `responds_with(:read, content)` into a private method, `def file_with(content)`.

Reply:
I've reduced the length of the line by moving the `.with` onto the next line.